scripts/update_template.pl

```diff
@@ -58,6 +58,9 @@ sub assemble
 		|| die $t->error(), "\n";
 }
 
+unless (-e $template && -r $template) {
+	die "JavaScript Challenge template '$template' not found, generation aborted!\n";
+}
```

Comment:
I think it's better to move the check higher, where we check all the arguments from `@ARGV`